### PR TITLE
Import Linux Kernel typo history

### PR DESCRIPTION
**Pull request by Gelma**
Hi everybody, I finished to import the biggest part of "fix typo" commit of Linux Kernel.

I merged in batch of 20 words for each commit.

Tell me if you prefer in another way.

Dividing by first letter get huge and tiny commits. Maybe in this way is much more manageable.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -54,6 +54,7 @@ abscence->absence
 abselutely->absolutely
 abselutly->absolutely
 absense->absence
+absolete->absolute
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: can also be `obsolete`

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -85,9 +86,12 @@ acccept->accept
 acccepted->accepted
 acccepting->accepting
 acccepts->accepts
+accces->access
 acccess->access
+acccessed->accessed
 acccesses->accesses
 acccessible->accessible
+acccessor->accessors
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: the plural here is wrong. `accessor` should be the solution

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -205,6 +222,7 @@ achor->anchor
 achored->anchored
 achoring->anchoring
 achors->anchors
+aci->acpi
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: not sure about this one, may return false positives

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -254,30 +273,41 @@ acqured->acquired
 acqures->acquires
 acquring->acquiring
 acrage->acreage
+acses->cases
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: can also be `access`

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -254,30 +273,41 @@ acqured->acquired
...
 acssumed->assumed
+actally->actually
 actaul->actual
 actaully->actually
 actibity->activity
 actine->active
+actiual->actual
+activ->active
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: not sure, even though this makes sense, there may be code that purposefully spells the word this way. So... it's debatable

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -353,11 +387,14 @@ adin->admin
 aditional->additional
 aditionally->additionally
 aditionaly->additionally
+adivce->advise
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: can also be `advice` (actually should be the first solution)

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -386,25 +424,32 @@ adresable->addressable
...
 advizable->advisable
+adwances->advances
 aeactivate->deactivate, activate,
 aequidistant->equidistant
 aequivalent->equivalent
 aeriel->aerial
 aeriels->aerials
 afair->affair
+afaraid->afraid
+afect->effect
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: 1st solution should be `affect`
https://en.oxforddictionaries.com/usage/affect-or-effect

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -497,14 +547,18 @@ alege->allege
 aleged->alleged
 alegience->allegiance
 alegorical->allegorical
+aleviating->alleviating
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: please also add `aleviate->alleviate`

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -1195,32 +1316,40 @@ attendents->attendants
...
 attribtes->attribute
+attribtue->attribute
+attribude->attribute
+attribue->attribute
 attribues->attributes
 attribuite->attribute
 attribures->attributes
 attribut->attribute
 attributei->attribute
+attributen->attriburen
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: should be `attribute`
note: thi may throw false positives for any german spelling

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -1290,15 +1423,19 @@ auxillary->auxiliary
...
 avaialble->available
 avaiblable->available
 avaible->available
 availabe->available
 availabel->available
 availabelity->availability
 availabilty->availability
+availabl->available
 availabled->available
+availablen->availbalen
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: should be `available`

**Comment by luzpaz** [comment]
The preferred way is to submit 1 letter at a time alphabetically, for PRs this long. 
Some PRs can include several letters to make them large enough. I've gone through the letter 'a' so far and posted my feedback above. I think you should turn this PR in to 'Import Linux Kernel typo history: letter 'a'' and move the other commits in to their separate PRs. That's what I've done in the past for large commits like this.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: I'd agree in this case, although accessors is also a valid spelling according to Wiktionary (although not Chrome's spellchecker) https://en.wiktionary.org/wiki/accessors#English

**Comment by Gelma** [comment]
Hi @luzpaz ,
  I'm going to do PRs divided by first letter.
  I keep open this PR to migrate suggestions later.


Thanks a lot,
Andrea

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: thanks a lot both for the review. Tell me how to procede.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Well, anyway I don't see any valid meaning for "aci", in general pourpose.
So, I should let it in, but tell me.

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: You tell me.

File: codespell_lib/data/dictionary.txt (hunk 7 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Ok. Anyway, what I do is to report the case I see, not what it could be.

**Comment by Gelma** [comment]
Thanks a lot @luzpaz  for your review.

I migrated the fixes for letter A in its own branch.

So we can continue there.

If you agree, you can close this pull request.

Thanks a lot,
Andrea

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Surely put it in, as it's a typo, and they should blacklist it if they want to purposefully misspell it?

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Assuming this is https://en.wikipedia.org/wiki/ACPI then can you capitalise the RHS please.

**Pull request closed without merging**